x/oracle/types/msgs_test.go

```diff
@@ -91,7 +91,7 @@ func TestMsgAggregateExchangeRateVote(t *testing.T) {
 	msgInvalidSalt := "invalid salt length; must be 64"
 	msgInvalidOverflowValue := "out of range; bitLen:"
 	msgInvalidHexString := "salt must be a valid hex string: invalid salt format"
-	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: unknown request"
+	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: invalid request"
```

Comment:
This was the only test issue

Reply:
I think Yes, It is changed from `Unkown Request` to `Invalid Request` https://github.com/umee-network/umee/pull/1788/files/7909e6d8d5a98eb5763a92b2a30b4903da69e776#diff-0947889d44c2e808b3b719383dd20fd8282c8152705b41c27763555593b82029L114-R114